part-08.mdx

```diff
@@ -0,0 +1,230 @@
+# Build your first custom dashboard
+
+In previous parts of the tutorial, you have learned how several sections of Netdata dashboard worked. 
+
+This guide will show you how to easily create custom pages and set up a custom dashboard to fit your custom 
+or specific needs.
+
+## What you'll learn in this part
+
+In this part of the Netdata guide, you'll learn how to:
+
+-   create a custom-dashboard.html file
+-   Add dashboard.js
+-   Set the default Netdata server
+-   Create a chart, set its duration and size
+-   Set dimensions for your custom chart
+
+Let's get on with it!
+
+## Create a custom-dashboard.html file
+
+You can create your dashboard using simple HTML; in some cases for basic dashboards, 
+you may not need Javascript.
+
+On the same custom dashboard, you can make use of any available chart libraries ([Google Chart](https://developers.google.com/chart/), [D3.js](http://d3js.org/), and so on).
+
+You can also use data from one or any number of Netdata servers on the same dashboard as well as 
+host your dashboard HTML page on any web server.
+
+You can also add Netdata charts to existing web pages.
+
+In your local Netdata installation (http://myhost:19999/dashboard.html), you can find a custom dashboard template 
```

Comment:
This sentence gets a little long. Maybe try this as an alternative?

> In your local Netdata installation (`http://myhost:19999/dashboard.html`), you can find a custom dashboard template, which contains samples of all the supported chart types. This can help you get a better idea of how to put together your custom dashboard.